microsoft-edge/devtools-guide-chromium/overview.md

```diff
@@ -155,6 +155,30 @@ See [Keyboard shortcuts](shortcuts/index.md) and [Navigate DevTools with assisti
 You can dock DevTools in the right, left, or bottom of the browser, or undock it into a separate window.  See [Change DevTools placement (Undock, Dock to bottom, Dock to left)](customize/placement.md).
 
 
+<!-- ====================================================================== -->
+## Zooming DevTools or the rendered page
+
+The DevTools UI is implemented using HTML and CSS, and you can zoom it in and out like any other web page.  The zoom level for the DevTools portion of the browser is independent from the zoom level for the rendered webpage portion of the browser.
+
+There are 17 zoom levels: 8 zoom-out levels + 1 standard level (100%) + 8 zoom-in levels.  When you zoom the rendered page, the zoom percentage level is briefly displayed.
+
+
+To zoom the rendered page:
+
+1. If focus isn't already on the rendered page, click somewhere in the rendered page.
+1. Press `Ctrl`+`+` or `Ctrl`+`-` (Windows or Linux).  Or, press `Command`+`+` or `Command`+`-` (macOS).
+
+![Browser zoomed out, DevTools zoomed in.](overview-images/zoom-rendered-page.png)
```

Comment:
This documentation is about DevTools, so explaining how to zoom the rendered page doesn't make a lot of sense here I think. It's definitely ok to mention that both DevTools and the page can be zoomed with the same keyboard shortcuts, but I wouldn't necessarily add steps and a screenshot to explain how to do it in the page.

How about merging the 2 sub-sections like this:

```
To zoom the DevTools portion of the browser:

1. If focus isn't already on DevTools, click somewhere in DevTools.
1. Press `Ctrl`+`+` or `Ctrl`+`-` (Windows or Linux).  Or, press `Command`+`+` or `Command`+`-` (macOS).

![Browser zoomed in, DevTools zoomed out.](overview-images/zoom-devtools.png)

Alternatively, if you want to zoom the rendered page, click somewhere in the rendered page and use the same keyboard shortcuts.
```